ports/discordcoreapi/vcpkg.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "discordcoreapi",
+  "version": "1.0.2",
```

```suggestion
  "version-date": "2022-03-30",
```
If you are using a single commit rather than a release, please use version-date.
The value of the version-date is the date you submitted this PR.